### PR TITLE
feat!: computed members and audit log config added

### PR DESCRIPTION
**Pull request by waxb**
- chore: changelog updated
- chore: update module from template and tests
- feat!: computed members and audit log config added

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -25,8 +25,19 @@ variable "members" {
...
+  }
+}
+
+variable "computed_members_map" {
+  type        = map(string)
+  description = "(Optional) A map of members to replace in 'var.members' or in members of 'policy_bindings' to handle terraform computed values."
+  default     = {}
+
+  validation {
+    condition     = alltrue([for k, v in var.computed_members_map : can(regex("^(allUsers|allAuthenticatedUsers|(user|serviceAccount|group|domain|computed):)", v))])
```
Comment (mariux) [inline variables.tf:39]:
```suggestion
    condition     = alltrue([for k, v in var.computed_members_map : can(regex("^(allUsers|allAuthenticatedUsers|(user|serviceAccount|group|domain):)", v))])
```

File: variables.tf [hunk 2]
```diff
@@ -25,8 +25,19 @@ variable "members" {
...
+    error_message = "The value must be a non-empty string being a valid principal type identified with `allUsers`, `allAuthenticatedUsers` or prefixed with `user:`, `serviceAccount:`, `group:`, `domain:` or `computed:`."
```
Comment (mariux) [inline variables.tf:40]:
```suggestion
    error_message = "The value must be a non-empty string being a valid principal type identified with `allUsers`, `allAuthenticatedUsers` or prefixed with `user:`, `serviceAccount:`, `group:`, or `domain:`."
```

File: variables.tf [hunk 3]
```diff
@@ -25,8 +25,19 @@ variable "members" {
   default     = []
 
   validation {
-    condition     = alltrue([for m in var.members : can(regex("^(user|serviceAccount|group|domain):(.+)", m))])
-    error_message = "The value must be a non-empty list of strings where each entry is a valid principal type identified with a prefix such as e.g., `user:`, `serviceAccount:`, `group:` or `domain:`."
+    condition     = alltrue([for m in var.members : can(regex("^(allUsers|allAuthenticatedUsers|(user|serviceAccount|group|domain|computed):)", m))])
+    error_message = "The value must be a non-empty list of strings where each entry is a valid principal type identified with `allUsers`, `allAuthenticatedUsers` or prefixed with `user:`, `serviceAccount:`, `group:`, `domain:` or `computed`."
```
Comment (mariux) [inline variables.tf:29]: i don't think allusers and allauthenticated is valid here... where did you get this from?

**Review by mariux — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 3 above)
Comment (waxb) [reply variables.tf:29]: https://github.com/mineiros-io/terraform-google-kms-key-ring-iam/blob/33428a62eddd07d60002b2a771311cc83ae0578a/variables.tf#L22

the oirigin :)

Comment (mariux) [reply variables.tf:29]: yes but this is a different resource... with different features ofc ;) it wa snot there before.. so i  assume it is no available in this resource?

**Review by mariux — CHANGES_REQUESTED** [review]
requesting same changes again.

**Review by zied-elouaer — DISMISSED** [review]
LGTM!

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -7,15 +7,8 @@ on:
...
-env:
-  AWS_SECRET_ACCESS_KEY: ${{ secrets.MINEIROS_TESTING_AWS_SECRET_ACCESS_KEY }}
-  AWS_ACCESS_KEY_ID: ${{ secrets.MINEIROS_TESTING_AWS_ACCESS_KEY_ID }}
-  GOOGLE_CREDENTIALS: ${{ secrets.MINEIROS_TESTING_GCP_SA_KEY_FILE }}
-  TEST_GCP_PROJECT: ${{ secrets.MINEIROS_TESTING_GCP_PROJECT }}
-  TEST_GCP_ORG_DOMAIN: ${{ secrets.MINEIROS_TESTING_GCP_ORG_DOMAIN }}
-
 concurrency:
-  group: terraform-google-project-iam
+  group: terraform-module-template
```
Comment (mariux) [inline .github/workflows/tests.yml:11]: adjustments from template are missing.

File: variables.tf (hunk 3 above)
Comment (mariux) [reply variables.tf:29]: @waxb why was this resolved but not addressed?

**Review by mariux — CHANGES_REQUESTED** [review]

**Review by mariux — APPROVED** [review]
lgtm... thx ;)

**Pull request merged**